lambdas/sqs-message-remover/tests/test-index.js

```diff
@@ -233,3 +241,38 @@ test('sqsMessageRemover lambda updates message visibilityTimeout when workflow f
 
   await awsServices.sqs().deleteQueue({ QueueUrl: sqsQueues.queueUrl }).promise();
 });
+
+test.serial('sqsMessageRemover lambda removes message from S3 when workflow succeeded', async (t) => {
+  process.env.system_bucket = randomString();
+  process.env.stackName = randomString();
+  await createBucket(process.env.system_bucket);
+
+  const sqsQueues = await createSqsQueues(randomString());
+  const msgBody = JSON.stringify({ testdata: randomString() });
+
+  await awsServices.sqs().sendMessage({
+    QueueUrl: sqsQueues.queueUrl, MessageBody: msgBody,
+  }).promise();
+  const sqsOptions = { numOfMessages: 10, visibilityTimeout: 120, waitTimeSeconds: 20 };
+  const receiveMessageResponse = await receiveSQSMessages(sqsQueues.queueUrl, sqsOptions);
+  const { MessageId: messageId, ReceiptHandle: receiptHandle } = receiveMessageResponse[0];
+
+  const eventSource = createEventSource({ messageId, receiptHandle, queueUrl: sqsQueues.queueUrl });
+  const eventMessage = createCloudwatchEventMessage({
+    status: 'SUCCEEDED',
+    eventSource,
+  });
+  const key = `${process.env.stackName}/archived-incoming-messages/${eventSource.messageId}`;
+
+  await s3PutObject({
+    Bucket: process.env.system_bucket,
+    Key: key,
+    Body: msgBody,
+  });
+
+  await updateSqsQueue(eventMessage);
+  await t.throwsAsync(s3().getObject({
```

Comment:
Nit: prefer `getObject` from `aws-client/s3`

Reply:
Like this? [`b43c09c` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/b43c09c2e91fbdf652ce65d88a0bb760a3df8639) I don't know if I prefer it but don't feel too strongly.